#include <stdexcept>

class Undo : public std::exception {
};